<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Multimedia processing tool</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotatePart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate, Scaling
Filp, Wrap, Crop
Part</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImgePart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>740</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image
Part</string>
    </property>
   </widget>
   <widget class="QPushButton" name="histogramPart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram
Part</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterPart">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enhance image quality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_view">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="result_view">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="webCamBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Web Cam
START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yolov5Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Counting App</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1396</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
